lighthouse-viewer/types/viewer.d.ts

```diff
@@ -37,6 +39,14 @@ declare global {
     // Inserted by viewer gulpfile build.
     LH_CURRENT_VERSION: string;
   }
+
+  interface PSIParams {
```

Comment:
if there's no typescript-magic needed or it's not used in many different files, a lot better just to define in `psi-api.js` and pull into the `lighthouse-report-viewer.js` (like with `Settings`)

Reply:
oh, whoops, more of a viewer PR comment :)

Reply:
Using JSDoc to define interfaces really sucks. anything longer than like, 3 properties  has to be broken into multiple lines, which makes a lot of red in VSCode  (for me, maybe there's a better plugin?)

Reply:
just use the multiline typedef, e.g. https://github.com/GoogleChrome/lighthouse/blob/e4a1e1aa9b975e3436fafa3923130048d6853f1b/lighthouse-core/scripts/i18n/bake-ctc-to-lhl.js#L16-L22

(and globals suck worse)